### ISSUE TITLE
Pageless mode

### ISSUE THREAD

**Issue by ScanuNicco** [adil192/saber#88]
It would be awesome if there were a pageless mode, allowing infinite scrolling in every direction. (sort of like OneNote)

**Comment by ceskyDJ** [comment]
That is completely true. I have one point here – please allow to turn on/off this feature for every note separately. Sometimes I need to have pages, sometimes don't.

**Comment by iamashwin99** [comment]
I would also like this feature

**Comment by thiswillbeyourgithub** [comment]
I am also in dire need of this feature. Is there a bounty system available? I would gladly participate with $$ and I prefer bounties to undirected donations.

**Comment by thiswillbeyourgithub** [comment]
Could it be implemented as a temporary workaround to allow larger pages and/or square pages? That might be way simpler to implement and actually provide enough functionnality for some of us for some time. Even standard A1 paper would be much better!

I'm willing to bounty a few bucks if that helps :)

**Comment by adriy-be** [comment]
I approve, to be able to increase page size would be interesting.
Why ? When I import a PDF I can't add some information around the PDF, I must create a new page and put my note on the new page ...

**Comment by Lygaen** [comment]
Hi !

Is there any to check/help progress on this ? Maybe marticipate as well !

I saw that there is a branch related to this, is it okay to use or not yet ready ?

Thx !

**Comment by QubaB** [comment]
I created pull request #1487. Vertical space can be inserted to page and entities below point of insertion are moved down (page is eventually enlarged).  

This method can be used to enlarge page to "almost" infinite. It can be easily extended to also resize page width.

**Comment by fabioromeo** [comment]
This would be an infinite canvas, and I'm in a dire need for it. Some notes fit nice in a page format, but others works more like a brainstorm, and the ability to have notes expand to every direction would make my life incredibly easier. Some times I go back to previous ideas and notes on previous pages just to add observations, drawings or questions, and I often run out of space when doing that.

**Comment by thiswillbeyourgithub** [comment]
That and the cognitive load of "where in the page should I start" is really hurting my flow.

**Comment by QubaB** [comment]
You can try to compile your own Saber version with my pull request #1487. It can be used to enlarge page (now it enlarges only verttical) but it can be easily extended.

**Comment by fabioromeo** [comment]
Any updates on this feature? It's on top of the list for some time now.
